tests/saitests/py3/sai_qos_tests.py

```diff
@@ -2298,7 +2300,7 @@ def runTest(self):
                                     ecn=ecn,
                                     ttl=ttl)
             dst_port_2_id = self.get_rx_port(
-                src_port_id, pkt_dst_mac, dst_port_2_ip, src_port_ip, dst_port_2_id, src_port_vlan
+                src_port_id, pkt_dst_mac2, dst_port_2_ip, src_port_ip, dst_port_2_id, src_port_vlan
```

Comment:
for t0, if router_mac is empty, pkt_dst_macX should be dst_port_X_mac, then it should work.
but for t1, router_mac must not be empty since all ports are routing interfaces. in that case, pkt_dst_macX will share the same value among different ports. will the logic be broken in this case?

Reply:
No, tests pass with this fix also on t1 topology, also this fix is included in 202305 Xon test code, this is not new code to the test.